cpu/nrf5x_common/periph/pm.c

```diff
@@ -35,4 +41,5 @@ void pm_off(void)
 #endif
     NRF_POWER->SYSTEMOFF = 1;
     while(1) {}
```

Comment:
Ah sorry 
```suggestion
    while (1) {}
```
was fine